src/MonsterImage.tsx

```diff
@@ -22,6 +22,7 @@ export function MonsterImage({
   scale = 1,
 }: MonsterImageProps): JSX.Element {
   const [state, setState] = React.useState<State>("loading");
+  const [shiny, setShiny] = React.useState<boolean>(false);
```

Comment:
you can write `useState(false)` instead here; TypeScript is able to infer the type `boolean` based on the parameter :)

Reply:
Ohhh, yeah I saw that it worked without the type but wasn't sure if it was best practice.

Reply:
I think that if TypeScript is able to infer the exact same type you're specifying, you should omit it. Generally gonna be true for string/number/boolean